### DISCUSSION TITLE
Trouble with #split on ish app Linux shell for iPad OS

### DISCUSSION THREAD

**Discussion by fal3nsh** [scandum/tintin#77] [Bugs]
Hi there,

I'm having trouble using #split in a proper way with **ish Linux shell on my iPad Pro**, in comparison with what I get on a regular desktop macos terminal with tintin++.

Basic #split (or #split 0 0) seems to work, as an input bar is isolated and does what it is supposed to do. But whenever I try to get more advanced #split, like #split 3 1, the client doesn't respond properly. Box and line are drawn, at first, showing the dedicated "zones", but with any input on the screen (lines received, #showme, etc.), the split vanishes and the lines, or text, or event #prompt captures sent to the default bar, then scroll with the entire text upward. It seems also that the buffer of in-game texte received is truncated... missing entire parts.

Any workaround or help to get ish app to work properly on this ? It seems to me like it is related to client, not tintin, but I shall ask the question here at first :) Thank you.

**Comment by scandum** [comment]
This sounds like a bug in iSH. Best to reach out to its developer.

I'm quite sure split used to work, so it might be a recent change.

**Reply by fal3nsh** [reply]
Thanks, I'll drop a discussion on ish github as well. I'll post any outcomes here.

**Comment by fal3nsh** [comment]
I've found a neat solution to be able to play from ish on ipad: just install tmux on ish (apk add tmux), start tmux server (tmux), launch tt++ from the window 0 created (and optionnaly hide the status bar for "full screen" back again). Then you can #split tintin as you want with correct scrolling, display maps perfectly, etc. Hope this helps someone else :)

**Comment by scandum** [comment]
Odd, but possible that tmux alters some of the VT100 codes used by tintin.

Might still be worth bringing it up with iSH devs since tintin's use of VT100 is a-typical but valid.

Out of curiosity, does #split 3 1 1 work correctly?

**Comment by fal3nsh** [comment]
Yes, #split 3 1 1 appears to work perfectly as well on tmux (left pane is displayed and do not "scroll" like it would don under iSH not muxed). I've reported the issue to the iSH devs, but had no answer for now.
Thanks !
